tools/driver/CMakeLists.txt

```diff
@@ -26,6 +26,7 @@ if(BOOTSTRAPPING_MODE MATCHES "BOOTSTRAPPING.*")
     HAS_SWIFT_MODULES
     BOOTSTRAPPING 0
     THINLTO_LD64_ADD_FLTO_CODEGEN_ONLY
+    REQUIRED_FOR_MINIMAL_COMPILER
```

Comment:
Could skip adding this and the next, since they're only used for bootstrapping which we're not doing.